hellochain/cmd/hcd/main.go

```diff
@@ -4,7 +4,7 @@ import (
 	"github.com/tendermint/tendermint/libs/cli"
```

Comment:
why was this done? these utils are specific to hellochain and have nothing to do with the nameservice tutorial?
everything tutorial specific goes with the tutorial not at the root